README.md

```diff
@@ -88,6 +89,22 @@ You can run the test server with `cd packages/kit/test/apps/basics; pnpm run dev
 
 You may need to install some dependencies first, e.g. with `npx playwright install-deps` (which only works on Ubuntu).
 
+To test Vite locally, add the below code into the root `package.json` and run `pnpm install` to link the local Vite package.
+
+```jsonc
+{
+	// ...
+	"dependencies": {
+		"vite": "^2.7.1"
```

Comment:
I think `2.0.0` might make more sense so that 2.7.1 doesn't seem like an arbitrary version listed here after there have been some new Vite releases
```suggestion
		"vite": "^2.0.0"
```